packages/core/src/utils/errorWrapper.ts

```diff
@@ -1,4 +1,4 @@
-import { ERROR_CODE, ERROR_CODE_DESC, ERROR_MESSAGE } from '@/constants';
+import { ERROR_CODE, ERROR_CODE_DESC, ERROR_MESSAGE } from '../constants/errors';
```

Comment:
`@constants/errors`?

Reply:
Unfortunately can't use, because this file is going to banner bundle too.

But only right now I thought that it would be better to put it in common dir, do u agree?

Reply:
Yeah, let's do it

Reply:
done